PATCH-P0-HACK-SUSE: Make abi_check pass

Because of our gcc settings, abi_check fails when it doesn't find all includes.
This comments them out in the case of abi_check. So we do abi_check, but only
on these things that matters and doesn't prevent us from actually doing it.

Maintainer: Michal Hrusecky <Michal.Hrusecky@opensuse.org>

Index: include/mysql/client_plugin.h
===================================================================
--- include/mysql/client_plugin.h.orig
+++ include/mysql/client_plugin.h
@@ -23,8 +23,10 @@
 */
 #define MYSQL_CLIENT_PLUGIN_INCLUDED
 
+#ifndef MYSQL_ABI_CHECK
 #include <stdarg.h>
 #include <stdlib.h>
+#endif
 
 /* known plugin types */
 #define MYSQL_CLIENT_reserved1               0
Index: include/mysql/client_plugin.h.pp
===================================================================
--- include/mysql/client_plugin.h.pp.orig
+++ include/mysql/client_plugin.h.pp
@@ -1,5 +1,3 @@
-#include <stdarg.h>
-#include <stdlib.h>
 struct st_mysql_client_plugin
 {
   int type; unsigned int interface_version; const char *name; const char *author; const char *desc; unsigned int version[3]; const char *license; void *mysql_api; int (*init)(char *, size_t, int, va_list); int (*deinit)(); int (*options)(const char *option, const void *);
